### ISSUE TITLE
GitHub is broken because of using ECMAScript2020/2021 features, Seamonkey/UXP polyfill extension available

### ISSUE THREAD

**Issue by CeruleanSky** [MrAlex94/Waterfox-Old#70]
**Describe the bug**
GitHub is broken because of using ECMAScript2020/2021 features
![image](https://user-images.githubusercontent.com/2829875/153694991-4df093a9-8d17-4716-8fab-a6b77e983f92.png)
https://webcache.googleusercontent.com/search?q=cache:https://github.blog/changelog/2022-02-10-deperecation-notice-for-ecmascript2020-support/

**To Reproduce**
1. Go to github
2. Pr Controls, seeing timestamps relatively, other functionality is broken

**Expected behavior**
The site to function

**Desktop (please complete the following information):**
All versions of Classic on all OSes

**Additional context**
Discussion:
https://forum.palemoon.org/viewtopic.php?f=70&t=27879&p=224118
https://github.com/JustOff/github-wc-polyfill/issues/43

Polyfill Fix extension that needs to be adapted to work with Waterfox:
https://github.com/JustOff/github-wc-polyfill

**Comment by kswartz26** [comment]
FWIW, I installed version 1.2.11b1 of the github-wc-polyfill extension, and most of the functionality (including timestamps and tag browsing from the drop-down, both of which recently broke) seems to be working now. The "Go to File" search still does not work for me though.

**Comment by CeruleanSky** [comment]
Go to File may be related to #14 , there is a polyfill there for the function, but it is not in addon form.

**Comment by kswartz26** [comment]
> Go to File may be related to #14 , there is a polyfill there for the function, but it is not in addon form.

You're right. The first solution there, tucked into a Greasemonkey script, fixes that one. Thanks for the pointer!

**Comment by CeruleanSky** [comment]
Glad it worked out for you, mind sharing your Greasemonkey script so that others can use it as well?

Also the github-wc-polyfil does contain a replaceChildren function, but I guess it isn't the right type or has some issue with Waterfox.  
Perhaps you could consider doing a pull request over at github-wc-polyfill's repo there so a working alternative is included.

**Comment by reallyuniquename** [comment]
Addon github-wc-polyfil doesn't support e10s.

**Comment by Squall-Leonhart** [comment]
> 

Polly does however.

At the expense of some serious CPU usage when first loading the page, and when typing into the text box.
